### PR TITLE
36 improvement services cards - grid /style

### PR DESCRIPTION
**Pull request by Roxha-V**
Fixed grid and card padding for small devices. Also i had to reinstall swiper library due the library was not functioning correctly and the app wasnt finding the modules.

### CODE REVIEW

File: app/components/contact_section/index.jsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
 "use client";
+
 function ContactComponent() {
   return (
-    <>
+    <div>
```
Comment (devPalter) [inline app/components/contact_section/index.jsx]: cambio que no acorde a la tarea

File: package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "next": "14.1.0",
     "react": "^18",
     "react-dom": "^18",
-    "swiper": "^11.1.3"
+    "swiper": "^11.1.4"
```
Comment (devPalter) [inline package.json]: cambio no acorde a la tarea

**Review by devPalter — CHANGES_REQUESTED** [review]

**Comment by Roxha-V** [comment]
resuelto y comiteado . services cards review improvements and coments were resolved (grid /padding/shadow/styles)

**Comment by Roxha-V** [comment]
rebase de la rama hecho  pero muestra la siguiente rta : remote: - 1 review requesting changes by reviewers with write access. [Requiere revisión de propietarios de código: Se necesita una revisión de al menos un propietario de código para los cambios en la rama develop.]

**Review by devPalter — APPROVED** [review]

**Pull request merged**